frontend/src/features/public-form/components/FormPaymentPage/stripe/StripeElementWrapper.tsx

```diff
@@ -114,7 +113,10 @@ const StripePaymentContainer = ({
         return (
           <>
             <PaymentSuccessSvgr maxW="100%" />
-            <StripeReceiptContainer formId={formId} paymentId={paymentId} />
+            <StripeReceiptContainer
+              formId={formId}
+              paymentId={paymentInfoData.submissionId}
```

Comment:
Should we not change the name of the property in `StripeReceiptContainer` too?

Reply:
you're right, thanks! done in [a447277](https://github.com/opengovsg/FormSG/pull/6116/commits/a4472776a6ccf698431708deba2c4f4e2e1c0593)